examples/app.py

```diff
@@ -110,4 +110,4 @@ def current_user():
 '''
 if __name__ == '__main__':
     app.run(threaded=True, port=int(os.environ.get("PORT",
-                                                   os.environ.get("SPOTIPY_REDIRECT_URI", 8080).split(":")[-1])))
+                                                   os.environ.get("SPOTIPY_REDIRECT_URI".split(":")[-1], 8080))))
```

Comment:
Hmm it's now doing `.split()` directly on the `SPOTIPY_REDIRECT_URI` string?

    "SPOTIPY_REDIRECT_URI".split(":")

What problem did you try to solve? Please add a description and a changelog to your PR next time.